src/Adapter/Image/ImageRetriever.php

```diff
@@ -212,6 +213,11 @@ public function getImage($object, $id_image)
 
         // Check and generate each thumbnail size
         $image_types = ImageType::getImagesTypes($type, true);
```

Comment:
Maybe we should add a `$theme = null` parameter in the `getImagesTypes` method?

Reply:
And when `$theme` is left null we return ALL the image types

Reply:
Totally true! It's fixed.